.jenkins/pytorch/macos-test.sh

```diff
@@ -16,11 +16,12 @@ fi
 export PATH="${PYTORCH_ENV_DIR}/miniconda3/bin:$PATH"
 source ${PYTORCH_ENV_DIR}/miniconda3/bin/activate
 conda install -y mkl mkl-include numpy pyyaml setuptools cmake cffi ninja six
-pip install hypothesis librosa>=0.6.2 psutil
+pip install -q hypothesis "librosa>=0.6.2" psutil
```

Comment:
quotes are needed on macos and rocm builds otherwise it leaves a file named `>=0.6.2` behind (wtf).